Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright Contributors to the Open Cluster Management project
+FROM golang:1.19 AS builder
```

Comment:
may use `registry.ci.openshift.org/stolostron/builder:go1.19-linux AS builder`

Reply:
I have switched the base image from `golang:1.19` to `registry.ci.openshift.org/stolostron/builder:go1.19-linux`